.github/workflows/public-ci.yml

```
@@ -0,0 +1,302 @@
+        if: matrix.target == 'x86_64-unknown-linux-gnu'
+        run: |
+          python3 .github/scripts/rusty_v8_bazel.py check-module-bazel
+          just test-github-scripts
+      - name: Check MODULE.bazel.lock is up to date
+        if: matrix.target == 'x86_64-unknown-linux-gnu'
+        run: ./scripts/check-module-bazel-lock.sh
+      - name: bazel test //...
+        env:
+          BUILDBUDDY_API_KEY: ${{ secrets.BUILDBUDDY_API_KEY }}
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:13:31: Double quote to prevent globbing and word splitting

(shellcheck)